aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,103 @@
+import { FeedbackAnatomy, SimpleFeedbackExample, SolicitedFeedbackExample } from '../../examples';
+import { Example } from '../../layouts';
+import { Checkmark } from 'grommet-icons';
+
+export const contact = {
+  name: 'Doris Singer',
+  email: 'doris.singer@hpe.com',
```

Comment:
Given this is a public facing site, just confirming we are fine putting Doris's email directly here? I know we've done this with the HPE Global Header team's general email but just checking.

Reply:
Good catch. I'd recommend removing and maybe placing a link to something internal that has her contact info?

Reply:
checking with Doris on this